Drivers/seds/PioHwas/example/work/hwas/implem/default/C/src/hwas.c

```diff
@@ -1,12 +1,23 @@
-/* Body file for function HWAS
- * Generated by TASTE on 2021-11-29 11:46:27
- * You can edit this file, it will not be overwritten
- * Provided interfaces : InterruptManagement_DisableInterrupt_Pi, InterruptManagement_EnableInterrupt_Pi,
- * InterruptSubscriptionManagement_SubscribeToInterrupt_Pi, RawMemoryAccess_ReadBuffer_Pi, RawMemoryAccess_ReadWord_Pi,
- * RawMemoryAccess_WriteBuffer_Pi, RawMemoryAccess_WriteWord_Pi Required interfaces : InterruptSubscription_Interrupt_Ri
- * User-defined properties for this function:
- *   |_ TASTE::Is_Component_Type = false
- * Timers              :
+/**@file
+ * This file is part of the TASTE Runtime SAMV71.
+ *
+ * @copyright 2022 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime SAMV71 was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
```

Comment:
I just wanted to remove the incorrect boilerplate. The license for this is GPL BTW :)